DSCResources/MSFT_HybridRunbookWorker/MSFT_HybridRunbookWorker.psm1

```diff
@@ -311,7 +311,21 @@ function TestRegModule
 #>
 function TestRegRegistry
 {
-    Test-Path -Path HKLM:\SOFTWARE\Microsoft\HybridRunbookWorker
+    $Reg = Get-ItemProperty -Path HKLM:\SOFTWARE\Microsoft\HybridRunbookWorker
```

Comment:
```Get-ItemProperty -Path HKLM:\SOFTWARE\Microsoft\HybridRunbookWorker``` throws an error as you can see [here](https://ci.appveyor.com/project/bgelens/hybridrunbookworkerdsc/build/0.0.0.23#L940). It is not terminating but as it is expected up front, it would be better if it was suppressed with ```-ErrorAction SilentlyContinue```

Reply:
Thanks Ben. Updated the function to return true or false based on whether the user hybrid worker is registered. 
Thanks for writing this resource.
Eamon